Feat/component tab textbox speechbubble#220

hyuns66 opened the pull request:
## #️⃣연관된 이슈

- #220 

## 📝작업 내용

- tabRow 컴포넌트 추가
- textbox 컴포넌트 추가
- coderabbit 설정 파일 추가
- typo 변경사항 반영
- 이전 PR 컨플릭트 수정

## PR 발행 전 체크 리스트

- [ ] 발행자 확인
- [ ] 프로젝트 설정 확인
- [ ] 라벨 확인
- [ ] 코드 린트 확인

## 스크린샷 (선택)

## 💬리뷰 요구사항(선택)

- 리뷰어가 특별히 봐주었으면 하는 부분이 있다면 작성해주세요
  - speech bubble은 좀 까다로워서 아직 안했어용

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* 신기능
  * 검색바 입력/지우기 콜백 및 기본값 지원 추가, 새 텍스트 박스(AtchaTextBox) 컴포넌트 도입.
  * 1버튼/2버튼 다이얼로그와 커스텀 버튼 문구 지원 추가.

* 개선/버그수정
  * 알람 볼륨 제어 추가(재생 시 설정 적용, 종료 시 원복).
  * 디버그용 잠금화면 알람 시간 조정.

* 디자인/스타일
  * 대규모 타이포그래피 토큰 교체 및 다수 컴포넌트 스타일 업데이트.
  * 스플래시 로고 및 배경 변경, 말풍선·탭바 등 시각 개선.

* 변경/제거
  * 홈 인사 바텀시트 제거 및 일부 탭/컴포넌트 위치 변경.
  * 공용 구성 파일(.coderabbit.yaml) 추가 및 문자열 리소스 보강.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

hyuns66 committed:
feat/#177 : LockService -> 알람 채널 사용하도록 수정

- 기존에는 미디어 볼륨으로 알람이 울려서 사용자가 임의로 알람소리 크기를 조절할 수 있던 이슈가 있었습니다.
- 그로인해 의도치 않게 알람이 울리지 않을 수 있었습니다.
- 알람이 울릴 때 미디어 볼륨이 아니라 알람 채널을 활용하도록 하여 항상 같은 크기로 알람이 울릴 수 있도록 수정했습니다. `

hyuns66 committed:
feat/#177 : 알람 볼륨 설정하는 repository 매서드 추가

- 나중에 마이페이지에서 설정하는 UI 에 반영해야 합니다.

hyuns66 committed:
fix/#220 : 일부 typo 변경사항 반영

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble

hyuns66 committed:
fix/#220 : text style 변수명 변경

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble

hyuns66 committed:
feat/#220 : tabRow 컴포넌트화

hyuns66 committed:
feat/#220 : textbox 컴포넌트화

hyuns66 committed:
feat/#220 : TextBox 컴포넌트 주석처리

hyuns66 committed:
chord : code rabbit 설정파일 추가

hyuns66 committed:
chore : linting

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble#220

hyuns66 committed:
Merge branch 'develop' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble#220

jieeeunnn commented on app/src/main/java/com/depromeet/team6/ui/theme/Type.kt:
```diff
@@ -11,41 +11,37 @@ import com.depromeet.team6.R
+    val body3_B3M15: TextStyle,
+    val body4_B4R15: TextStyle,
+    val body5_B5SB14: TextStyle,
     val bodyMedium14: TextStyle,
-    val bodyRegular14: TextStyle,
+    val body6_B6R14: TextStyle,
     val bodySemiBold13: TextStyle,
-    val bodyMedium13: TextStyle,
+    val body7_B7M13: TextStyle,
     val bodyRegular13: TextStyle,
     val bodySemiBold12: TextStyle,
     val bodyMedium12: TextStyle,
-    val bodyRegular12: TextStyle,
+    val detail1_R12: TextStyle,
     val bodySemiBold11: TextStyle,
-    val bodyMedium11: TextStyle,
+    val detail2_M11: TextStyle,
     val bodyRegular11: TextStyle,
     val bodySemiBold10: TextStyle,
-    val bodyMedium10: TextStyle,
+    val detail3_M9: TextStyle,
     val bodyRegular10: TextStyle,
```
타이포 네이밍도 디쟌팀에서 변경된 건가요??

hyuns66 replied:
넵
타이포 아예 싹다 갈아엎어져서 지금 하나하나 적용하고 있어요 ㅋㅋㅋㅋㅋ
좀이따 PR 올릴건데 새로생긴 typo 기준으로 작업해주셔야 합니다!

jieeeunnn reviewed (approved):
우왕 토끼 친구 생겼다 고생하셨습니다!

jieeeunnn committed:
feat/#223 : 공통 컴포넌트 SearchBar 수정

jieeeunnn committed:
feat/#223 : 공통 컴포넌트로 수정하면서 불필요해진 컴포넌트 삭제

jieeeunnn committed:
feat/#223 : home strings 추가

jieeeunnn committed:
feat/#223 : HomeScreen 검색 부분 공통 컴포넌트 적용

jieeeunnn committed:
chore/#223 : ktlint 적용

hyuns66 committed:
feat/#220 : speech bubble 컴포넌트화

- 벡터 표현식 (createOutline 매서드) 에서 radius 그리는 과정에 2배 작게 그려지는  오류가 있어서 수정했습니다.
- 말풍선 꼬리는 못고치겠네염

hyuns66 committed:
Merge branch 'feat/component-tab-textbox-speechbubble#220' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble#220

hyuns66 committed:
feat/#220 : homeGreetBottomSheet 삭제

- 기획 내용에서 사라져서 삭제했습니다.

t1nm1ksun committed:
feat/#227: 다이얼로그 State 분기처리

t1nm1ksun committed:
feat/#227: OneButtonDialog 구현

t1nm1ksun committed:
chore/#227: ktlintFormat

t1nm1ksun committed:
chore/#230: OnboardingSearchLocationItem 패딩값 수정

t1nm1ksun committed:
chore/#230: 스플래시 앱 로고 변경

t1nm1ksun committed:
chore/#230: ktlintFormat

t1nm1ksun committed:
feat/#227: 다이얼로그 텍스트 명시

hyuns66 committed:
fix/#220 : typo 관련 변경사항 전부 반영 완료

- 디자인시스템에서 네이밍 변경됐고 일부 UI에 폰트 스타일도 변경됐습니다.
- typo 관련 수정하면서 대부분 수정하긴 했는데 모든 UI 다 꼼꼼히 체크하진 못해서 디자인시스템 작업하면서 한번씩 UI 확인해주세요

hyuns66 committed:
Merge branch 'feat/home-component#223' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble#220

hyuns66 committed:
Merge branch 'feat/component-minseok' of https://github.com/depromeet/16th-team6-Android into feat/component-tab-textbox-speechbubble#220

hyuns66 committed:
fix/#220 : 코드리뷰 반영

hyuns66 committed:
chore : code rabbit 프롬프트 추가
